recipes/polylaue/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "polylaue" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PolyLaue/PolyLaue/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 9205ae1df3daadf3727ca5adb3a3ecb553c02e8e0f01e87487aa5f9bea7ef490
+
+build:
+  skip: true  # [py<310]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - fabio
+    - h5py
+    - numba
+    - numpy
+    - pillow
+    - platformdirs
+    - pyqtgraph
+    - pyside6
+    - python
+    - scipy
+
+test:
+  imports:
+    - {{ name }}
```

Comment:
Please don't use jinja outside of the bare minimum above. It obscures the recipe and doesn't add any value.